app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainRepositoryImplement.java

```diff
@@ -0,0 +1,28 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+/**
+ * Created by lapnguyen on 16/06/2017.
+ */
+import com.example.lapnguyen.weather_3.data.Weather;
+import java.util.List;
+import java.util.concurrent.Callable;
```

Comment:
ERROR: Unused import: java.util.concurrent.Callable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports